pinot-tools/src/main/java/org/apache/pinot/tools/admin/command/QuickstartRunner.java

```diff
@@ -77,6 +75,7 @@ public class QuickstartRunner {
   private final List<Integer> _controllerPorts = new ArrayList<>();
   private final List<Integer> _brokerPorts = new ArrayList<>();
   private boolean _isStopped = false;
+  public static final int DEFAULT_CONTROLLER_PORT = 9000;
```

Comment:
Can you move this line back to its original location so that we keep all port constants together?

Reply:
Done! I was keeping public methods at the end but now know the reason from the other PR you reviewed. :)